### ISSUE TITLE
[CI] Add a "prefer-lowest" build on Travis

### ISSUE THREAD

**Issue by mbabker** [Sylius/SyliusResourceBundle#184]
It looks like all of the Travis builds are testing with the latest versions of dependencies within whatever constraints are used on a build (i.e. the different Symfony versions or with #182 different versions of the state machine packages).  Is there a reason there isn't a build that passes the `--prefer-lowest` flag into Composer to force a build with the minimum declared dependencies?  In part, this helps ensure the minimums are correctly defined.

**Comment by loic425** [comment]
I don't think we have to do that. This #182 has been fixed now and #187 (depending on it) works too.

**Comment by loevgaard** [comment]
I agree with @mbabker on this one. If you don't have a build with lowest dependencies, you can't say for certain that your library works with the constraints in your `composer.json`.

**Comment by loic425** [comment]
Ok, I misread that. I read we have to use "--prefer-lowest" on each build.
